package.json

```diff
@@ -35,16 +35,19 @@
   },
   "homepage": "https://github.com/frostney/rollup-plugin-alias#readme",
   "devDependencies": {
-    "ava": "^0.12.0",
-    "coveralls": "^2.11.8",
-    "eslint": "^2.2.0",
-    "eslint-config-airbnb": "^6.0.2",
-    "npm-run-all": "^1.5.1",
-    "nyc": "^5.6.0",
+    "ava": "^0.25.0",
+    "coveralls": "^3.0.2",
+    "eslint": "^5.10.0",
+    "eslint-config-airbnb": "^17.1.0",
```

Comment:
This might be out of scope, but can't use `airbnb-base` as we don't really need the react config and `jsx-a11y`

Reply:
I changed the ESLint config to match more or less that of Rollup